codec-stomp/src/main/java/io/netty/handler/codec/stomp/DefaultStompHeadersSubframe.java

```diff
@@ -24,13 +24,19 @@
 
     protected final StompCommand command;
     protected DecoderResult decoderResult = DecoderResult.SUCCESS;
-    protected final StompHeaders headers = new DefaultStompHeaders();
+    protected final DefaultStompHeaders headers;
```

Comment:
@amizurov can you explain why the changes to this class are needed at all ? I think you can revert the changes

Reply:
So, when we create `new DefaultStompFrame` in `DefaultStompFrame #replace(...)` method we pass headers.copy() to the constructor.

Reply:
But not to this class

Reply:
It's super class for `DefaultStompFrame`, else it's will be look like this: 
 ```java
DefaultStompFrame(StompCommand command, ByteBuf content, DefaultStompHeaders 
headers)  {
        super(command);

        if (headers != null) {
            this.headers.set(headers);
        }

        if (content == null) {
            throw new NullPointerException("content");
        }
        this.content = content;
    }

Reply:
never mind missed it is the super-class.